src/Microsoft.Android.Sdk.ILLink/PreserveLists/Mono.Android.Runtime.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<linker>
+	<assembly fullname="Mono.Android.Runtime">
```

Comment:
Should we just say "preserve this entire assembly"?  Is there a way to say that?